libraries/commerce/filter/actions/getFilters.js

```diff
@@ -36,7 +32,7 @@ const getFilters = () => (dispatch, getState) => {
   }
 
   // We need to process the params to handle edge cases in the pipeline params.
-  const requestParams = processParams(params, activeFilters);
+  const requestParams = processParams(params, null);
```

Comment:
Why does this function accept a second parameter that is always null?

Reply:
I've removed the `null` but set it as a default value for the 2nd parameter of `processParams`. I added null because the `getActiveFilters` returned `null` when no active filters where present.